test/src/test/java/org/corfudb/runtime/object/transactions/OptimisticTransactionContextTest.java

```diff
@@ -123,9 +125,14 @@ public void customConflictObjectsConflictAborts()
      */
     @Test
     public void checkSlowWriterTxAbortsOnHoleFill() {
+        final Duration holeFillTimeout = Duration.ofSeconds(1);
```

Comment:
what is that rational of 1 second? Maybe add that ass comment in case that we need to change it in the future.

Reply:
Done.